x-pack/test/functional_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -13,68 +13,71 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const testSubjects = getService('testSubjects');
 
   // FLAKY: https://github.com/elastic/kibana/issues/63621
-  describe.skip('host list', function () {
+  describe.skip('endpoint list', function () {
```

Comment:
should this still be skipped?

Reply:
@kevinlog yes. Although I updated the tests with new paths, etc. they are still "flaky" - @charlie-pichette is (will) working on getting them re-nabled. :)